tldr/dfg_extractor.py

```
@@ -1304,7 +1484,8 @@ def extract_c_dfg(code: str, function_name: str) -> DFGInfo:
 
 def _find_c_function_by_name(root, name: str, source: bytes):
     """Find a C function node by name in tree-sitter tree."""
-    def search(node):
+
+    def search(node) -> Any:
```

Ruff (0.14.13)

Dynamically typed expressions (typing.Any) are disallowed in `search`

(ANN401)